docs/Permissioning/Permissioning apis.md

```diff
@@ -0,0 +1,427 @@
+This api can be executed by the organization admin account to update the status of a node.
+
+* Input:  organization id or sub organization id, enode id, action (1 for deactivating the node, 2 for activating a deactivated node and 3 for blacklisting a node)
+* Output: Status of the operation
+* Example:
+```javascript
+> quorumPermission.updateNodeStatus("ABC.SUB1.SUB2.SUB3", "enode://239c1f044a2b03b6c4713109af036b775c5418fe4ca63b04b1ce00124af00ddab7cc088fc46020cdc783b6207efe624551be4c06a994993d8d70f684688fb7cf@127.0.0.1:21006?discport=0&raftport=50407",3, {from: eth.accounts[1]})
+{
+  msg: "Action completed successfully",
+  status: true
+}
+```
+Once a node is blacklisted no further action is possible on the same.
+
+### Roles
+
+#### Organization status types
+The table below indicates the numeric value for various organization statuses.
+
+| OrgStatus                 |           Value |
+| :-----------------------: | :-------------: |
+| NotInList                 |               0 |
+| Proposed                  |               1 |
+| Approved                  |               2 |
+| PendingSuspension         |               3 |
+| Suspended                 |               4 |
+| AwaitingSuspensionRevoke  |               5 |
+
+#### Account status types
+The table below indicates the numeric value for various account statuses.
+
+| AccountStatus   |           Value |
+| :-------------: | :-------------: |
+| NotInList       |               0 |
+| PendingApproval |               1 |
+| Active          |               2 |
+| Inactive        |               3 |
+| Suspended       |               4 |
+| Blacklisted     |               5 |
+| Revoked         |               6 |
+
+#### Account access types
+The table below indicates the numeric value for each account access type.
+
+| AccessType      |           Value |
+| :-------------: | :-------------: |
+| ReadOnly        |               0 |
+| Transact        |               1 |
+| Contract deploy |               2 |
+| Full access     |               3 |
+
+When setting the account access, the system checks if the account setting the access has sufficient privileges to perform the activity. 
+
+* Accounts with `FullAccess` can grant any access type ( FullAccess, Transact, ContractDeploy or ReadOnly) to any other account
+* Accounts with `ContractDeploy` can grant only `Transact`, `ContractDeploy` or `ReadOnly` access to other accounts
+* Accounts with `Transact` access can grant only `Transact` or `ReadOnly` access to other accounts
+* Accounts with `ReadOnly` access cannot grant any access
+
+#### Node Status types
+The table below indicates the numeric value for various node statuses.
+
+| NodeStatus                |           Value |
+| :-----------------------: | :-------------: |
+| NotInList                 |               0 |
+| PendingApproval           |               1 |
+| Deactivated               |               2 |
+| Blacklisted               |               3 |
```

Comment:
this list doesn't match with values defined in `core/types/permissions_cache.go`

Reply:
updated documentation.